src/crewai/flow/flow.py

```
@@ -1043,6 +1043,7 @@ async def _execute_single_listener(self, listener_name: str, result: Any) -> Non
             import traceback
 
             traceback.print_exc()
+            raise
```

no more silently failing